scenarios/appsec/test_ip_blocking.py

```diff
@@ -14,15 +14,14 @@
 @rfc("https://docs.google.com/document/d/1GUd8p7HBp9gP0a6PZmDY26dpGrS1Ztef9OYdbK3Vq3M/edit")
 @released(cpp="?", dotnet="2.16.0", php="?", python="?", ruby="?", nodejs="?", golang="?")
 @released(java={"spring-boot": "0.110.0", "sprint-boot-jetty": "0.111.0", "spring-boot-undertow": "0.111.0", "*": "?"})
+@irrelevant(context.appsec_rules_file == "")
```

Comment:
The context is handled by scenarios. So if a scenario does not set a rule file, it should not include this test class, so this decorator should be useless. Or do I miss something ?

Reply:
Same comment as on the other PR. If there is a rules file inside `binaries/`, `DD_APPSEC_RULES` will still be set.